kubernetes-manifests/shoppingassistantservice.yaml

```diff
@@ -42,7 +42,7 @@ spec:
             drop:
               - ALL
           privileged: false
-          readOnlyRootFilesystem: true
+          readOnlyRootFilesystem: false
```

Comment:
Why is this necessary? Ideally we don't need root access in our container as it's a security risk. None of the other services in Online Boutique needs it, so I'm curious what makes this one different?

Reply:
It was a result of a failure to write to the tmp directory when accessing alloyDB. I tried granting permissions and defining a tmp directory that wasn't on the root to get around this, but it still failed to write, so I took this workaround for the sake of time.